src/login.py

```diff
@@ -69,12 +71,12 @@ def executeLogin(self):
         try:
             self.enterPassword(self.browser.password)
         except Exception:  # pylint: disable=broad-except
-            logging.error("[LOGIN] " + "2FA Code required !")
+            logging.info("[LOGIN] " + "2FA Code required !")

```

Comment:
This should definitely still be a `.error` still to get the user's attention as it's important that 2fa is inputted.

Reply:
IMO error logging should be reserved for unrecoverable/unexpected situations from the app's perspective.

Asking for a 2FA code isn't really either of those. Either way, the user will know what's up because the logging will stop at this line. Pretty obvious what to do next by looking at the output.

Reply:
We should probably just print to the console since I can't imagine a situation where we'd ever not want to print this (i.e. if error logging or info logging is disabled)

Reply:
Good food for thought though @Titaniumtown

Reply:
> We should probably just print to the console since I can't imagine a situation where we'd ever not want to print this (i.e. if error logging or info logging is disabled)

Sounds good